docs/development/releasing.rst

```diff
@@ -1,233 +1,386 @@
-   a separate branch and open a pull request into the release branch to allow for
-   easy review. First, create and switch to a new branch based off the release
-   branch, e.g.::
+You may also want to locally run the tests (with remote data on to ensure all
+of the tests actually run), using tox to do a thorough test in an isolated
+environment::
 
-      $ git checkout -b v5.0-changelog
+   $ pip install tox --upgrade
+   $ TEST_READ_HUGE_FILE=1 tox -e test-alldeps -- --remote-data=any
 
-   Next, run towncrier and confirm that the fragments can be deleted::
+Additional notes
+----------------
 
-       towncrier --version 5.0
+Do not render the changelog with towncrier at this point. This should only be done just before the final
+release. However, it is up to the discretion of the release manager whether to
+open 'practice' pull requests to do this as part of the beta/release candidate
+process (but they should not be merged in) - if so the process for rendering the changelog is described
+in :ref:`render-changelog`.
 
-   Then add and commit those changes with::
+.. _release-procedure-tagging:
 
-      $ git add CHANGES.rst
-      $ git commit -m "Finalizing changelog for v<version>"
+Tagging the first release candidate
+-----------------------------------
 
-   Push to GitHub and open a pull request for merging this into the release branch,
-   e.g. v5.0.x.
+Assuming all the CI passes, you should now be ready to do a first release
+candidate! Ensure you have a GPG key pair available for when git needs to sign
+the tag you create for the release.  See :ref:`key-signing-info` for more on
+this.
 
-   In cases where an LTS branch and a different release branch are being maintained,
-   the changelog should be rendered on both branches separately, and only the
-   rendering from the non-LTS release branch should be forward-ported to main.
+Make sure your local release branch is up-to-date with the upstream release
+branch, then tag the latest commit with the ``-s`` option, including an ``rc1``
+suffix, e.g.::
 
-   .. note::
+      $ git tag -s v5.0rc1 -m "Tagging v5.0rc1"
 
-      We render the changelog on the latest release branch and forward-port it
-      rather than rendering on main and backporting, since the latter would
-      render all news fragments into the changelog rather than only the ones
-      intended for the e.g. v5.0.x release branch.
+Push up the tag to the `astropy core repository`_, e.g.::
 
-#. Once the changelog pull request is merged, update your release branch to
-   match the upstream version, then (on the release branch), tag the merge
-   commit for the changelog changes with ``v<version>``, being certain to sign
-   the tag with the ``-s`` option::
+      $ git push upstream v5.0rc1
 
-      $ git tag -s v<version> -m "Tagging v<version>"
+.. note::
+
+   It might be tempting to use the ``--tags`` argument to ``git push``,
+   but this should *not* be done, as it might push up some unintended tags.
```

Comment:
maybe put this into the red warning box instead of the note one :)

Reply:
Done